src/ui/common/src/components/workflows/WorkflowSettings.tsx

```diff
@@ -343,6 +357,12 @@ const WorkflowSettings: React.FC<WorkflowSettingsProps> = ({
           />
         </>
       )}
+
+      {triggerType === WorkflowUpdateTrigger.Cascade && (
+        <>
```

Comment:
Nit: If there's just one element you can do away with the fragment (<></>) that's wrapping it.